player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -45,7 +45,11 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
     };
    
     $scope.replayContent = function() {
-        EventBus.dispatch("event:openUserSwitchingModal", {'logGEEvent': $scope.pluginInstance._isAvailable});
+        if(isbrowserpreview) {
```

Comment:
Better to use `isMobile` check rather than `isBrowserPreview`  becoz `isbrowserPreveiw` is checking for the `webview` attribute in the url.

Suppose in the future if `webview` is got removed in the url parameter  then your functionality will going to break

Reply:
It should be configurable to show or hide the userswitch popup